util/winutil/winutil_notwindows.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !windows
+
+package winutil
+
+const RegBase = ``
```

Comment:
this isn't exported in the other file, so do we need this?

Reply:
It is now. I apparently forgot to save a file before committing.